### PR TITLE
Add user agent for requests.

### PR DESCRIPTION
**Pull request by jmoseley**
## Summary by CodeRabbit

- **New Features**
	- Updated the package version for the project to indicate new features or bug fixes.
	- Enhanced API client to include a dynamic `User-Agent` header for improved tracking and version management in HTTP requests.
- **Improvements**
	- Enabled JSON module resolution in TypeScript, allowing for seamless imports of JSON files, enhancing data handling capabilities.

### CODE REVIEW

File: src/api-client.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { version } from "../package.json";
```
Comment (jmoseley) [inline src/api-client.ts:1]: In theory this should work. I'm about 30% skeptical. We will test it once this gets published.

Comment (jmoseley) [reply src/api-client.ts:1]: Actually less skeptical now. It looks like `tsup` just hoists the value out of the `package.json` file when it bundles it, so when the library is installed and executed, this just refers to a constant.

image

File: src/api-client.ts [hunk 2]
```diff
@@ -38,6 +40,7 @@ export class CortexApiClient {
       method: "POST",
       headers: {
         Authorization: `Bearer ${this.accessToken}`,
+        "User-Agent": `cortex-js-sdk-v${version}`,
```
Comment (EvanBoyle) [inline src/api-client.ts:43]: I would encourage us to user something closer to the spec (`product/version`): https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Maybe `cortex-nodejs-sdk/v0.0.9`? 

having the `/` also makes parsing a little bit easier in various contexts like sql/analytics.

**Review by EvanBoyle — APPROVED** [review]

File: src/api-client.ts (hunk 2 above)
Comment (jmoseley) [reply src/api-client.ts:43]: good thinking

**Pull request merged**